src/js/page_sections/RelatedLinks.js

```diff
@@ -9,7 +9,7 @@ const RelatedLinks = ({ relatedLinks, style, sectionTitle, sectionText, sectionL
 
   return (
     <div className={`coa-section ${style === 'primary' ? '' : 'coa-section--grey' }`}>
-      <div className="wrapper">
+      <div className="container-fluid wrapper">
```

Comment:
I deleted this component in my branch -- so no need for the change.
actually now that Im saying it, I'll just handle the conflict depending on merge order. Just a heads up depending on who merges first.